File: occupancy-manager-java/src/main/java/com/occupancy/manager/dto/OccupancyUsageRequest.java
package com.occupancy.manager.dto;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
*
* @author Ramachandra
*
*/

@Setter
@Getter
@ToString
public class OccupancyUsageRequest {

	@JsonProperty("Free Premium Rooms")
	private Integer premiumRooms;
	@JsonProperty("Free Economy Rooms")
	private Integer economyRooms;
	@JsonProperty("Payments")
	private List<Integer> payments;
	@JsonProperty("Minimum Premium")
	private Integer minimumPremium;
}

File: occupancy-manager-kotlin/src/main/kotlin/com/occupancy/manager/dto/OccupancyUsageRequest.kt
package com.occupancy.manager.dto

import com.fasterxml.jackson.annotation.JsonProperty

/**
 *
 * @author Ramachandra
 */
data class OccupancyUsageRequest(
	@JsonProperty("Free Premium Rooms")
	var premiumRooms: Int,
	@JsonProperty("Free Economy Rooms")
    var economyRooms: Int,
	@JsonProperty("Payments")
	var payments: List<Int>,
	@JsonProperty("Minimum Premium")
	var minimumPremium: Int,
)